src/viam/config/viam-cpp-sdkConfig.cmake.in

```diff
@@ -1,12 +1,18 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
+
```

Comment:
We need to propagate the new dependencies to consumers. I wish this happened automatically, but it doesn't. I took me quite a bit of digging to figure out why the example cmake project couldn't find the xtensor "library" until I remembered to do this.